e107_plugins/hero/languages/English/English_global.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
++----------------------------------------------------------------------+
+|        e107 website content management system Hungarian Language File
+|        Released under the terms and conditions of the
+|        GNU General Public License (http://gnu.org).
+|        Last Modified: 2020/12/18 12:42:44
+|
+|        $Author: Yesszus $
++----------------------------------------------------------------------+
+*/
+
+define("LAN_PLUGIN_HERO_DIZ", "Image and text slider, animated bullet points for the hero area of your home page.");
```

Comment:
@yesszus This file is correct. Only LANs used globally. (such as "all of the admin area" or "all of frontend") use the non-numeric naming standard.